oidc.go

```diff
@@ -31,6 +35,10 @@ const (
 	ScopeOfflineAccess = "offline_access"
 )
 
+// ErrNoAccessTokenHash is returned when the caller attempts to validate an
+// access token hash against an id token that doesn't carry a value for it.
+var ErrNoAccessTokenHash = errors.New("id token did not have an access token hash")
```

Comment:
Make this private? Instead of users comparing a sentinel error, we should probably just have them do something like

```
if idToken.AccessTokenHash == "" {
     // handle case
}
```